include/xgboost/context.h

```diff
@@ -74,6 +109,12 @@ struct DeviceOrd {
         return DeviceSym::CPU();
       case DeviceOrd::kCUDA:
         return DeviceSym::CUDA() + (':' + std::to_string(ordinal));
+      case DeviceOrd::kSyclDefault:
+        return DeviceSym::SYCL_default() + (':' + std::to_string(ordinal));
+      case DeviceOrd::kSyclCPU:
+        return DeviceSym::SYCL_CPU() + (':' + std::to_string(ordinal));
```

Comment:
What is "sycl:cpu:ordinal"

Reply:
It means using CPU with the specific index for multi CPU systems.

Reply:
multi-socket or multi-core?

Reply:
multi-socket